.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: "Lint test"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  app:
+    name: "Lint checking app"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

Comment:
does it make sense to already bump up to Node 18 ?

Reply:
I don't think it will affect anything, but I would like to do that in v3 branch where we also remove deprecated code rather than here.